frappe/utils/pdf.py

```diff
@@ -14,6 +14,13 @@ def get_pdf(html, options=None, output = None):
 	html, options = prepare_options(html, options)
 	fname = os.path.join("/tmp", "frappe-pdf-{0}.pdf".format(frappe.generate_hash()))
 
+	options.update({
+		"disable-external-links": "",
```

Comment:
we will need external + internal links for images

Reply:
Done